frontend/src/pages/org/workflow-detail.ts

```diff
@@ -1069,28 +1069,21 @@ export class WorkflowDetail extends LiteElement {
                 ></sl-icon>
                 ${msg("Replay Latest Crawl")}</sl-button
               >
-            `,
-          )}
-          ${when(
-            this.isCrawler,
```

Comment:
The check for `${when(this.isCrawler` must still be applied for the QA Latest Crawl button as viewers don't have access to this page.

Reply:
Very good point! Updated